syntax = "proto3";

package grpc_service;

service fileServices {
    rpc list(list_req) returns (list_res){}
    rpc search(search_req) returns (search_res){}
}

message list_req {}

message list_res {
    string method = 1;
    repeated string files = 2;
}

message search_req {
    string file_name = 1;
}

message search_res {
    string method = 1;
    string file_to_find = 2;
    string status = 3;
    string path = 4;
}   